pkg/scalers/activemq_scaler.go

```diff
@@ -0,0 +1,268 @@
+package scalers
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	v2beta2 "k8s.io/api/autoscaling/v2beta2"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	kedautil "github.com/kedacore/keda/v2/pkg/util"
+)
+
+type activeMQScaler struct {
+	metadata   *activeMQMetadata
+	httpClient *http.Client
+}
+
+type activeMQMetadata struct {
+	managementEndpoint string
+	destinationName    string
+	brokerName         string
+	username           string
+	password           string
+	restAPITemplate    string
+	targetQueueSize    int
+	metricName         string
+	scalerIndex        int
+}
+
+type activeMQMonitoring struct {
+	MsgCount  int   `json:"value"`
+	Status    int   `json:"status"`
+	Timestamp int64 `json:"timestamp"`
+}
+
+const (
+	defaultTargetQueueSize         = 10
+	defaultActiveMQRestAPITemplate = "http://<<managementEndpoint>>/api/jolokia/read/org.apache.activemq:type=Broker,brokerName=<<brokerName>>,destinationType=Queue,destinationName=<<destinationName>>/QueueSize"
+)
+
+var activeMQLog = logf.Log.WithName("activeMQ_scaler")
```

Comment:
@melisatanrverdi do you think it's worth passing a `logr.Logger` into the below `NewActiveMQScaler` constructor? I think some other scalers do what you did here, I'm just wondering your opinion. I can propose this change in a follow-up issue.

Reply:
I've used activeMQ log because I saw other scaler do that way. But I don't think it's worth passing into the NewScaler constructor.

Reply:
Yeah, let's keep it this way for now.